packages/airnode-deployer/src/cli/commands.ts

```diff
@@ -91,6 +94,17 @@ export async function deploy(configPath: string, secretsPath: string, receiptFil
   fs.rmSync(tmpDir, { recursive: true });
 
   writeReceiptFile(receiptFile, mnemonic, config, output, deploymentTimestamp);
+
+  if (deploymentError) {
+    logger.fail(
+      bold(
```

Comment:
I've added it to `bold` because this is important, but the message can be in the middle of the error log. See:

<img width="821" alt="image" src="https://user-images.githubusercontent.com/22679154/165145097-aae19c31-308d-43a3-b2ae-01f19feb4bc3.png">